gettsim/benefits/arbeitsl_geld.py

```diff
@@ -22,16 +22,20 @@ def ui(
         soli_st_params,
         beit_bem_grenz=soz_vers_beitr_params[f"rvmaxek{westost}"],
         werbungs_pausch=eink_st_abzuege_params["werbungskostenpauschale"],
-        soz_vers_pausch=params["soz_vers_pausch"],
+        sozial_versich_pauschale_alg=params["sozial_versich_pauschale_alg"],
```

Comment:
Identifier for social insurances is `soz_vers` and for `pauschale` there is `pausch` everywhere else. Also we use `arbeitsl_geld`.